app/src/main/java/de/fraunhofer/fokus/OpenMobileNetworkToolkit/MainActivity.java

```diff
@@ -351,6 +354,10 @@ public void onRequestPermissionsResult(int i, @NonNull String[] strArr, @NonNull
                         .turnOnCloudCityLoggingAfterPermissionsGranted(
                                 SharedPreferencesGrouper.getInstance(getApplicationContext())
                         );
+
+                // GPSMonitor
+                gpsMonitor = new GPSMonitor(getApplicationContext());
+                gpsMonitor.startMonitoring();
```

Comment:
pull this into an extension method in the MainActivityExtensions class, don't just leave it lying around here

Reply:
done